README.md

```diff
@@ -139,7 +136,7 @@ _Note: it's also possible to tunnel the connection to your own machine and use o
 # you can get this from the portal, or determine it via the inputs your pass to this module in your code
 POSTGRES_HOST=rpe-draft-store-aat.postgres.database.azure.com
 
-ssh -N bastion-dev-nonprod.platform.hmcts.net -L 5440:${POSTGRES_HOST}:5432
+ssh bastion-nonprod.platform.hmcts.net -L 5440:${POSTGRES_HOST}:5432
```

Comment:
why did you remove this?
```suggestion
ssh -N bastion-nonprod.platform.hmcts.net -L 5440:${POSTGRES_HOST}:5432
```

Reply:
-N switch doesn't seem to work for me so figured it wasn't necessary - could be just my ssh playing up